src/views/index.html

```diff
@@ -6,6 +6,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.0.0-alpha/js/bootstrap.min.js"></script>
+  <link rel="shortcut icon" href="https://dd7g6vmdl9i96.cloudfront.net/a1h0yvvqyf/images/favicon_128x128.png">
```

Comment:
Sorry I forgot that before ... can you copy the favicon into the local repo (we delete the old versions of files from time to time)